kernel/build/config-amd64

```diff
@@ -4992,7 +4992,17 @@ CONFIG_SYNC_FILE=y
 # end of DMABUF options
 
 CONFIG_DCA=y
-# CONFIG_UIO is not set
+CONFIG_UIO=m
```

Comment:
would `UIO` autoload `UIO_PCI_GENERIC` since UIO has PCI id udev should load it automatically, `UIO_PCI_GENERIC` doesn't seem to have pci id's so i'm not sure

Reply:
I tried to build just with `CONFIG_UIO=m` but `uio_pci_generic.ko` wasn't build.

Reply:
for the context, we unbind and rebind the nvme using `initContainer` 

```
echo $device > /sys/bus/pci/devices/$device/driver/unbind
echo "uio_pci_generic" > /sys/bus/pci/devices/$device/driver_override
echo $device > /sys/bus/pci/drivers/uio_pci_generic/bind
```

Reply:
> I tried to build just with `CONFIG_UIO=m` but `uio_pci_generic.ko` wasn't build.
> 
> ```
> _out/lib/modules/6.6.43-talos/kernel/drivers/uio
> _out/lib/modules/6.6.43-talos/kernel/drivers/uio/uio.ko
> ```

now : 
```
_out/lib/modules/6.6.45-talos/kernel/drivers/uio
_out/lib/modules/6.6.45-talos/kernel/drivers/uio/uio.ko
_out/lib/modules/6.6.45-talos/kernel/drivers/uio/uio_pci_generic.ko
```

Reply:
so `uio` is require by `uio_pci_generic`